pkg/specgenutil/specgen.go

```diff
@@ -592,10 +592,11 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 		s.DependencyContainers = c.Requires
 	}
 
-	// TODO
-	// outside of specgen and oci though
-	// defaults to true, check spec/storage
-	// s.readonly = c.ReadOnlyTmpFS
+	// Only add ReadWrite tmpfs mounts iff the container is container is
+	// being run ReadOnly and ReadWrite tmpfs are disabled,
```

Comment:
- You have `container is` twice
- Shouldn't this be `ReadWrite tmpfs is not disabled`?

Reply:
Yup, hopefully no more typos...